containers/datalab/content/setup-env.sh

```diff
@@ -15,12 +15,14 @@
 
 # Sets up various environment variables within the docker container.
 
-export DATALAB_USER=`gcloud -q config list --format yaml | grep account | awk -F" " '{print $2}'`
-export DATALAB_PROJECT_ID=`gcloud -q config list --format yaml | grep project | awk -F" " '{print $2}'`
-if [ -z $DATALAB_PROJECT_NUM ]; then
-  export DATALAB_PROJECT_NUM=`curl --silent -H "Metadata-Flavor=Google" http://metadata.google.internal/computeMetadata/v1beta1/project/numeric-project-id`
+if [ -z "$GOOGLE_APPLICATION_CREDENTIALS" ] && [ -z "$GCLOUDSDK_CONFIG" ]
+then
+  export CLOUDSDK_CONFIG=/content/datalab
```

Comment:
Not really. If the auth is done by the node server and we write the auth token ourselves, we want to write it in a place that doesn't clobber the users credential file outside the container. So putting it in datalab folder is fine. But the user may want to use their external credential file to do auth and stop the node server from doing so, so they may want to point it at /content/.config.